gdquest_art_tools/Infrastructure.py

```diff
@@ -39,7 +39,7 @@ def meta(self):
         meta = filter(lambda m: m[0] in self.cfg['meta'].keys(), meta)
         meta = OrderedDict((k, v.lower().split(s)) for k, v in meta)
         meta.update({k: map(int, v) for k, v in meta.items() if k in 'ms'})
-        meta.setdefault('c', self.cfg['meta']['c']) # coa_tools
+        meta.setdefault('c', self.cfg['meta']['c'])  # coa_tools
```

Comment:
For people with slight OCD

Reply:
Yeah, these little things tend to glitch when using a vanilla text editor and completely forget about doing any linting / formatting :stuck_out_tongue: